lib/ansible/module_utils/network/iosxr/argspec/interfaces/interfaces.py

```diff
@@ -0,0 +1,47 @@
+#
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#############################################
+#                WARNING                    #
+#############################################
+#
+# This file is auto generated by the resource
+#   module builder playbook.
+#
+# Do not edit this file manually.
+#
+# Changes to this file will be over written
+#   by the resource module builder.
+#
+# Changes should be made in the model used to
+#   generate this file or in the resource module
+#   builder template.
+#
+#############################################
+"""
+The arg spec for the ios_interfaces module
+"""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+class InterfacesArgs(object):
+
+    def __init__(self, **kwargs):
+        pass
+
+    argument_spec = {'config': {'elements': 'dict',
+                                'options': {'name': {'type': 'str', 'required': True},
+                                            'description': {'type': 'str'},
+                                            'enabled': {'default': True, 'type': 'bool'},
+                                            'speed': {'type': 'int', 'choices': [10, 100, 1000]},
```

Comment:
`speed` option might change with the iosxr series. Probably the choices can be removed and note can be added in speed option doc to refer to platform manual for accepted `speed` values

Reply:
sure, I'll remove the choice value and update the code if needed